libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -231,6 +233,90 @@ describe('ModalWrapperComponent', () => {
     });
   });
 
+  describe('with embedded page progress component', () => {
+    describe('with static page progress', () => {
+      beforeEach(() => {
+        spectator = modalWrapperTestBuilder
+          .flavor('modal')
+          .withStaticPageProgress()
+          .build();
+        spectator.detectComponentChanges();
+      });
+
+      afterEach(() => {
+        // Ensure any observers are destroyed:
+        spectator.fixture.destroy();
+      });
+
+      it('should move embedded page progress to wrapper component', () => {
+        const ionContentElement = spectator.query('ion-content');
+        const embeddedComponentElement = ionContentElement.firstElementChild;
+        const embeddedPageProgress = embeddedComponentElement.querySelector('kirby-page-progress');
+        const pageProgressAsWrapperChild = spectator.element.querySelector('kirby-page-progress');
+
+        expect(embeddedPageProgress).toBeNull();
+        expect(pageProgressAsWrapperChild).not.toBeNull();
+      });
```

Comment:
Align with other tests (you are not actually asserting the element was moved because of missing scope...)

Reply:
Point taken. See a21012b3